.changeset/afraid-oranges-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/aksel-icons": patch
+---
+
+Ikoner: LocationPin-ikoner
```

Comment:
(Småpirk) Selv om det teknisk sett er to forskjellige ikoner, så regnes det vel bare som ett ikon med to varianter, eller? Tenker på at "-ikoner" står i flertall. Tidligere har vi skrevet det på denne måten: "Ikoner: Nytt ikon `LocationPin`"